docs/design/w3c/spec.md

```diff
@@ -0,0 +1,530 @@
+  },
+  ...
+}
+```
+
+* `schema` - id of Indy Schema
+* `definition` - id of Indy Credential Definition
+* `revocation` - id of Indy Revocation Registry
+* `encoding` - attributes encoding algorithm
+    * encoded credential attribute values (binary representation required for doing CL signatures) are not included
+      neither to `credentialSubject` or `signature`
+    * `encoding: auto` implies using the algorithm defined
+      at [Aries RFC 0592 Indy Attachments section](https://github.com/hyperledger/aries-rfcs/tree/main/features/0592-indy-attachments#encoding-claims)
+      to generate encoded values under the hood during the signature generation and proof verification.
+
+#### Issuer
+
+W3C [Issuer](https://www.w3.org/TR/vc-data-model/#issuer) section requires including of `issuer` property to express the
+issuer of a verifiable credential.
+
+In the case of W3C AnonCreds credentials, the `issuer` attribute should be represented as a
+resolvable [DID URL](https://w3c-ccg.github.io/did-resolution/) having either `indy` or `sov` DID method and looks the
+following:
+
+```
+{
+  ... 
+  "issuer": "did:sov:3avoBCqDMFHFaKUHug9s8W",
+  ...
+}
+```
+
+#### Issuance Date
+
+W3C [Issuance Date](https://www.w3.org/TR/vc-data-model/#issuance-date) section requires including of `issuanceDate`
+property to express the date and time when a credential becomes valid.
+
+In the case of W3C AnonCreds credentials, for the `issuanceDate` attribute recommended setting of a random time of the
+day when credential was issued or transformed and looks the following:
+
+```
+{
+  ... 
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  ...
+}
+```
+
+#### Proofs (Signatures)
+
+W3C [Proofs (Signatures)](https://www.w3.org/TR/vc-data-model/#proofs-signatures) section requires including of `proof`
+property to express confirmation of the credential's validity.
+
+According to the specification, one or many proof objects can be added to verifiable credentials.
+In the context of W3C AnonCreds credentials included at least two proof object entries: AnonCreds CL
+and [Data Integrity](https://www.w3.org/TR/vc-data-model/#data-integrity-proofs).
+
+##### AnonCreds CL proof
+
+This proof entry derived from the CL signature of a verifiable credential.
+
+The defined `@context` includes a definition for the `CLSignature2022` type describing the format of the proof
+entry:
+
+```
+{
+  ... 
+  "proof": [
+    {
+      "type": "CLSignature2022",
+      "signature": "AAAgf9w5lZg....RYp8Z_x3FqdwRHoWruiF0FlM"
+    }
+  ]  
+  ...
+}
+```
+
+**Credential proof signature**
+
+* `type` - `CLSignature2022`
+* `signature` - credential signature
+    * signature received by building the following object from indy styled credential:
+      ```
+      {
+        "signature": {..}, 
+        "signature_correctness_proof": {..}
+      }
+      ```
+    * encoded
+      as [base64 attachment](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0017-attachments#base64url)
```

Comment:
base64-url-no-pad I assume instead of base64?

Reply:
Right. Basically, I just implied the reference to the RFC section here. Not exact format.